evergreen.yaml

```diff
@@ -167,67 +167,50 @@ task_groups:
 #           Buildvariants             #
 #######################################
 buildvariants:
-  - name: race-detector
-    display_name: Race Detector (Arch Linux)
-    expansions:
-      GOROOT: /opt/golang/go1.16
-      RACE_DETECTOR: true
-      SKIP_DOCKER_TESTS: true
-    run_on:
-      - archlinux-new-small
-      - archlinux-new-large
-    tasks:
-      - name: test_group
-
   - name: lint-linux
-    display_name: Lint (Arch Linux)
+    display_name: Lint (Ubuntu 22.04)
```

Comment:
Same nit as the other repos, just call this `Lint`.

Reply:
This is removed.

Reply:
Fixed